lib/Cargo.toml

```diff
@@ -25,6 +25,7 @@ thiserror = "1.0.0"
 tokio = { version = "1.5.0", features = ["full"] }
 
 [dev-dependencies]
+lenient_semver = { version = "0.4.2", default_features = false, features = ["version_lite"] }
```

Comment:
I see what you did there :D

Reply:
It's the proper crate for the use-case 🙃